app/lib/dttp/params/contact.rb

```diff
@@ -83,6 +88,17 @@ def trainee_not_disabled?
       def trainee_disability_not_provided?
         trainee.disability_disclosure == Diversities::DISABILITY_DISCLOSURE_ENUMS[:not_provided]
       end
+
+      def selected_nationality
+        # Handle edge case in the event a trainee somehow has no nationalities.
+        return CodeSets::Nationalities::BRITISH if trainee.nationalities.empty?
```

Comment:
Not sure if this edge case is needed as the validation around personal details makes sure at least one nationality is selected. And assumming "other" is removed from the list because there's no equivalent in DTTP (not to be confused by the "Other" checkbox in the UI), we can simplfy the code even more.

Reply:
Yeah it was just me being pedantic as you could technically run this job manually with a dodgy trainee created outside of those forms. I've removed it anyway